README.md

```diff
@@ -89,8 +89,9 @@ The following optional parameters can be set as a second argument in an array fo
 Parameter     | Default value                                  | Description
 ------------- | ---------------------------------------------- | -----------
 cookieName    | privacyCookieAccepted                          | Sets your own status cookie name
-days          | 365                                            | Says how many days privacy banner should be hidden when user accepts policy?
-bannerCaption | Cookies help us create a good experience (...) | Sets your own banner message caption
+cookieValidity          | 365                                            | Says how many days privacy banner should be hidden when user accepts policy?
```

Comment:
Good work @bedo-abn. Could you please adjust indents between `|` in this table?
`|` should be on the same position in every line - this makes everything more readable in plain text.